src/main/java/seedu/address/logic/commands/person/ViewGroupCommand.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.logic.commands.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.tag.IsTagInPersonPredicate;
+
+/**
+ * Command that lists all contacts who share the same tag
```

Comment:
Very small change that has to be made. Javadocs for classes should end with period.